config.example.ini

```diff
@@ -15,6 +15,9 @@
 #max_package_size = 20 MiB
 
 [worker]
+# Fully qualified name of the worker class
+#type = questionpy_server.worker.worker.subprocess.SubprocessWorker
```

Comment:
Ich denke, es wäre etwas nutzerfreundlicher, wenn man stattdessen direkt den Klassennamen angeben könnte (z.B.: `SubprocessWorker`).

Reply:
Würde halt voraussetzen, dass alle Worker-Impls in einem Dict oder im gleichen Modul zu haben sind.
So flexibel finde ich das schon besser, dann könnte die Worker-Impl bspw. auch in einem anderen Paket leben